front/tests/pages/common-page.ts

```diff
@@ -65,6 +71,11 @@ class CommonPage {
       })
     );
   }
+
+  async resourceNotFoundAssertion() {
```

Comment:
nit: to keep starting by a verb for methods and mmaybe reflect better what we expect ?
```suggestion
  async expectResourceNotFoundPage() {
```